iphone/Classes/WatchSessionModule.m

```diff
@@ -108,8 +153,20 @@ -(void)sendMessage:(id)value
         DebugLog(@"[ERROR] No watch paired");
         return;
     }
-    ENSURE_SINGLE_ARG(value, NSDictionary)
-    [[self watchSession] sendMessage:value replyHandler:nil errorHandler:nil];
+    ENSURE_SINGLE_ARG(args, NSDictionary)
+    NSDictionary *value = args;
+    KrollCallback *replyHandler = [value objectForKey:@"onReply"];
```

Comment:
Can we rename it to `reply` to keep the naming conventions like `success`, `error`and `cancel` callbacks?